test/spark/run-gc-test.sh

```diff
@@ -15,11 +19,35 @@ run_lakectl() {
 
 parallelism='-c spark.sql.shuffle.partitions=7 -c spark.default.parallelism=13 -c spark.hadoop.lakefs.gc.range.num_partitions=3 -c spark.hadoop.lakefs.gc.address.num_partitions=7'
 
-run_gc () {
-  if [ ${LAKEFS_BLOCKSTORE_TYPE} == "azure" ]; then
-    docker-compose run -v ${CLIENT_JAR}:/client/client.jar -T --no-deps --rm spark-submit bash -c "spark-submit -v --packages org.apache.hadoop:hadoop-azure:3.2.1 --master spark://spark:7077 --class io.treeverse.clients.GarbageCollector -c spark.hadoop.lakefs.api.url=http://docker.lakefs.io:8000/api/v1 -c spark.hadoop.lakefs.api.access_key=\${TESTER_ACCESS_KEY_ID} -c spark.hadoop.lakefs.api.secret_key=\${TESTER_SECRET_ACCESS_KEY} -c spark.hadoop.lakefs.api.connection.timeout_seconds=3 -c spark.hadoop.lakefs.api.read.timeout_seconds=8 -c spark.hadoop.fs.azure.account.key.\${LAKEFS_BLOCKSTORE_AZURE_STORAGE_ACCOUNT}.dfs.core.windows.net=\${LAKEFS_BLOCKSTORE_AZURE_STORAGE_ACCESS_KEY} $parallelism /client/client.jar $1"
+run_gc_according_to_storage_provider() {
+  local repo=$1
+  local additional_config_string=$2
+  if [[ ${LAKEFS_BLOCKSTORE_TYPE} == "azure" ]]; then
+    docker-compose run -v ${CLIENT_JAR}:/client/client.jar -T --no-deps --rm spark-submit bash -c "spark-submit -v --packages org.apache.hadoop:hadoop-azure:3.2.1 --master spark://spark:7077 --class io.treeverse.clients.GarbageCollector ${additional_config_string} -c spark.hadoop.lakefs.api.url=http://docker.lakefs.io:8000/api/v1 -c spark.hadoop.lakefs.api.access_key=\${TESTER_ACCESS_KEY_ID} -c spark.hadoop.lakefs.api.secret_key=\${TESTER_SECRET_ACCESS_KEY} -c spark.hadoop.lakefs.api.connection.timeout_seconds=3 -c spark.hadoop.lakefs.api.read.timeout_seconds=8 -c spark.hadoop.fs.azure.account.key.\${LAKEFS_BLOCKSTORE_AZURE_STORAGE_ACCOUNT}.dfs.core.windows.net=\${LAKEFS_BLOCKSTORE_AZURE_STORAGE_ACCESS_KEY} ${parallelism} /client/client.jar ${repo}"
```

Comment:
Can we split this line up using backslashes?  It's getting really hard to read (or I need a laptop with a 32x9 screen...).

Reply:
done, waiting for the tests to pass and pulling